website_menu_multilang/model/website.py

```diff
@@ -3,40 +3,32 @@
 # this module contains the full copyright notices and license terms.
 
 
-from openerp.osv import orm, fields
+from odoo import fields, models
 
 
-class WebsiteMenu(orm.Model):
+class WebsiteMenu(models.Model):
     _inherit = 'website.menu'
 
-    _columns = {
-        'lang_ids': fields.many2many(
-            'res.lang', 'website_menu_lang_rel', 'menu_id', 'lang_id',
-            'Allowed Languages',
-            help="Website languages this menu should be available for. "
-            "If kept empty, it will be available for all languages"
-        ),
-    }
-
-    def _default_lang_ids(self, cr, uid, context=None):
+    def _default_lang_ids(self):
         to_ret = []
-
-        website = self.pool.get('website').get_current_website(
-            cr, uid, context)
+        website = self.env['website'].get_current_website()
         if website and website.default_lang_id:
             to_ret = [(4, website.default_lang_id.id)]
 
         return to_ret
 
-    _defaults = {
-        'lang_ids': _default_lang_ids,
-    }
+    lang_ids = fields.Many2many(
+        'res.lang', 'website_menu_lang_rel', 'menu_id', 'lang_id',
+        'Allowed Languages',
+        help="Website languages this menu should be available for. "
+             "If kept empty, it will be available for all languages",
+        default=_default_lang_ids
```

Comment:
The common pattern is to declare fields on top, methods next.

Here you need then to use a `lambda self: self._default_lang_ids()` to be able to do that (which, in turn, is another requirement).